### DISCUSSION TITLE
Feature request about the UI

### DISCUSSION THREAD

**Discussion by dkkb** [outline/outline#2694] [Ideas]
I used to use Notion and I really like this project, here's some advice:

1. Creating new doc.
When I want to make a new document, I go to the 'Article' folder, move the mouse to the right side, and make a new document in two steps; if I want to make a nested document, I need to take another step.
Let's take a look at Notion, it displays the 'Create doc' menu button, it's simple and elegant.
Maybe it's a good idea to add `Create new doc` menu button. Because adding new doc is popular function, so put this button on the right is ideal.
<img src="https://user-images.githubusercontent.com/82504881/138870044-ef032857-f4c9-412d-9c8a-d7279ef710e5.png" width=200>
<img src="https://user-images.githubusercontent.com/82504881/138870119-d66e071f-6dc9-4987-bc90-9696845af2db.png" width=200>

2. Edit mode
People modify their documents frequently, therefore changing the view mode to edit mode is a crucial design decision. Actually, I appreciate the design in Notion, to change a document in Notion, simply click the doc area,  when you see the vertical cursor. OK, you wan work on it, that's simple. I really like this design. As we all know, the area of doc is large, so it's so easy to enter edit mode.

In outline, we must find the `Edit` button and click. Maybe many guys like writing in a single page, but I like the content sidebar, it displays an outline that helps me think clearly. So I think it's a good idea to edit the doc with the sidebar.

What are your thoughts about it?

Best wishes.

**Comment by tommoor** [comment]
1. We'll consider it, you can now also get to new document from `cmd+k` which is even faster

2. This has been a limitation of the underlying tech, we're in the process of moving everything to realtime collaborative editing – once that's enabled for everyone the "edit mode" becomes optional. There are also some key differences in the editor behavior between edit/non-edit that need normalizing too.

**Reply by dkkb** [reply]
1. But if I want to create a nested doc, `cmd+k` can't do this trick, right? 
2. I don't follow you, I think we can enter edit mode in a more straightforward manner, such as by clicking the position I want to modify, the edit mode maybe also display the sidebar and outline.

<img src="https://user-images.githubusercontent.com/82504881/139673008-b5267150-5e4d-4b4e-9a18-0e167c914970.png" width=400>

**Reply by tommoor** [reply]
1. It will work, but needs to be clearer – just use "create document" while within another.
2. This is being addressed here https://github.com/outline/outline/pull/2701
